user-operator/src/main/java/io/strimzi/operator/user/operator/KafkaUserOperator.java

```diff
@@ -290,6 +290,8 @@ public KafkaUserOperator(Vertx vertx,
         return CompositeFuture.join(secretOperations.reconcile(namespace, KafkaUserModel.getSecretName(user), null),
                 aclOperations.reconcile(KafkaUserModel.getTlsUserName(user), null),
                 aclOperations.reconcile(KafkaUserModel.getScramUserName(user), null),
+                kafkaUserQuotasOperator.reconcile(KafkaUserModel.getTlsUserName(user), null),
+                kafkaUserQuotasOperator.reconcile(user, null),
```

Comment:
Is deleting both just about being extra sure that there's nothing left? Because surely you know the type of user name in use, so you ought to be able to delete it using a single delete.

Reply:
Yeah, it is just about being sure. Should I leave there just the second option (`kafkaUserQuotasOperator.reconcile(user, null)`)?

Reply:
but in case of TLS user we have "CN=" as prefix, it's not just `user`, or?

Reply:
Mm, yes you are right.